README.md

```diff
@@ -1,5 +1,4 @@
-SMART FHIR Client
-=================
+# SMART FHIR Client
```

Comment:
We had a mix of underline and ## headers - consolidating on hashes, which are more obvious about what level you're at.